filebeat/_meta/common.full.p2.yml

```diff
@@ -215,6 +215,14 @@ filebeat.prospectors:
 # Configuration to use stdin input
 #- input_type: stdin
 
+#----------------------------- Redis prospector -------------------------------
+#- input_type: redis
```

Comment:
That is a tricky one. Thinking more about the prospectors I get the impression we will have prospector with multiple harvester types potentially (similar to what we have in metricbeat). For example in file there could be `file.csv` or `file.xml`. What if we not only have slowlog for redis? But TBH I'm not sure yet what we should do here. This could also well be overengineered and we will not need it.

For the moment I think I fix the above with call it `Redis slowlog prospector` in the docs but still `redis` as the prospector type.